### PR TITLE
Support for @defer query directive

### PR DESCRIPTION
**Pull request by patrys**
Proof of concept: missing docs, tests, and error support

### CODE REVIEW

File: ariadne/graphql.py [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
     GraphQLError,
     GraphQLSchema,
     TypeInfo,
-    execute,
     parse,
-    subscribe as _subscribe,
 )
+from graphql import subscribe as _subscribe
 from graphql.execution import MiddlewareManager
+from graphql.execution import execute as execute_sync
```
Comment (rafalp) [inline ariadne/graphql.py]:
```suggestion
from graphql.execution import MiddlewareManager, execute as execute_sync
```

**Comment by rafalp** [comment]
Looking good so far. We'll also need tests suite for this and entry in `CHANGELOG.md`

**Comment by squidsoup** [comment]
Is this still being worked on? @defer is a crucial pattern for implementing the graphql 1 view, 1 query pattern, particularly when that query has expensive fields.

**Comment by rafalp** [comment]
This work is currently on hold, but PR that would continue this work to completion (eg. rebase on master, optimize, add tests and error handling) is welcome.

**Comment by rafalp** [comment]
There's work happening on adding `@defer` to the `graphql-core` itself.

**Comment by levrik** [comment]
@rafalp It seems like `@defer` support is available in `graphql-core` nowadays. Is it working with Ariadne already?

**Pull request closed without merging**